src/cephadm/cephadm

```diff
@@ -3725,12 +3725,12 @@ def command_version(ctx):
 def command_pull(ctx):
     # type: (CephadmContext) -> int
 
-    _pull_image(ctx, ctx.image)
+    _pull_image(ctx, ctx.image, ctx.insecure)
```

Comment:
Please note that adding this to pull_image is actually not enough for real-world usage, as the run command might also pull the image. @guits found the same issue a few weeks back. Right now there is no way to fix this at this point as podman doesn't support `tls-verify=false` for the run command.

Reply:
right, but at least will allow us to pull an image before running it, which will pick up on the existing image in the system. :shrug:

Reply:
right. it's going to work most of the time.